litespeed-cache/var/crawlermap.data.meta

```diff
@@ -0,0 +1 @@
+{"list_size":7,"last_update_time":1565545079,"curr_crawler":0,"curr_crawler_beginning_time":1565545078,"last_pos":0,"last_count":1,"last_crawled":7,"last_start_time":1565545078,"last_status":"stopped","is_running":0,"end_reason":"Crawler #1 reached end of sitemap file.","meta_save_time":1565545079,"pos_reset_check":1565545083,"done":"touchedEnd","this_full_beginning_time":1565545078,"last_full_time_cost":1,"last_crawler_total_cost":1,"file_time":1565545078}
```

Comment:
Probably don't wanna have this in your PR

Reply:
Good call :o) thanks, need to add it to gitignore